.gitignore

```diff
@@ -8,6 +8,7 @@ cover.html
 integration.txt
 old.txt
 new.txt
+.idea
```

Comment:
Please check the comment at the top of this file.

Reply:
fixed, I missed it, thank you